drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -942,13 +942,15 @@ function DragAndDropBlock(runtime, element, configuration) {
         }
     };
 
-    var sliderInitializer = function() {
-        itemSlider = $root.find('.item-bank').bxSlider({
-            pager: false,
-            infiniteLoop: false,
-            startSlide: currentSlideIndex,
-            speed: 0
-        });
+    var initializeSlider = function() {
+        if (pageLoaded) {
+            itemSlider = $root.find('.item-bank').bxSlider({
```

Comment:
@jawad-khan  If we disable touch (like you did during testing), then this is at least usable on mobile, because click events are registered instead.  So I'd accept adding the following as a workaround, including a comment to explain why:
```javascript
// Workaround: mobile touch drag and drop behavior is broken,
// but disabling touch allows click events to come through.
touchEnabled: false,
```